.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
May i ask which convention this prettierrc follows?

Reply:
I thought I needed to set a consistent convention for efficient work, so I first applied the rule that I thought was most visible in other files. The other prettierrc files I added to the other libs are mostly the same.
Someday we should discuss about the right rules and apply it throughout the project.